tests/block/test_get_blocks.py

```diff
@@ -0,0 +1,328 @@
+    """
+    blocks_ids = 'fe56a16dab009cc96e7125c647b6c71eb1063818cf8dece283b125423ecb184f' \
+                 '7f1e61802bf66382da904698413f80831031f8a1b29150260c3fa4db537fdf4c, ' \
+                 '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+                 '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--ids',
+        blocks_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    list_of_blocks = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+    for block in list_of_blocks:
+        assert block.get('header_signature') in blocks_ids
+
+
+def test_get_blocks_invalid_ids():
+    """
+    Case: get a list of blocks by invalid identifiers.
+    Expect: the following identifier are not valid error message.
+    """
+    invalid_block_identifier = 'fe56a16dab009cc96e7125c647b6c71eb1063818cf8dece283b125423ecb184f'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--ids',
+        invalid_block_identifier,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'ids': [
+                f'The following block identifier `{invalid_block_identifier}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_blocks_with_limit():
+    """
+    Case: get a list of blocks limiting by a number.
+    Expect: specified number of blocks are returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--limit',
+        2,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    list_of_blocks = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert len(list_of_blocks) < 3
+
+
+def test_get_blocks_with_invalid_limit():
+    """
+    Case: get a list of blocks limiting by an invalid number.
+    Expect: the following limit should be a positive error message.
+    """
+    invalid_limit = -33
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--limit',
+        invalid_limit,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'limit': [
+                'Limit must be greater than 0.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_blocks_with_invalid_node_url():
```

Comment:
This test is for a non-existing node URL, but not an invalid node URL. Create test for invalid node URL and update documentation.
```suggestion
def test_get_batch_status_with_non_existing_node_url():
```

Reply:
@anastasiia-bilova, done.